src/mmw/sass/components/_modals.scss

```diff
@@ -31,7 +31,7 @@
 .modal-basic {
 
   .modal-content {
-    padding: 1rem;
+    padding: 3rem;
```

Comment:
The first screenshot is maybe taken from the "Share" button and the second from the "make public" (or the other way around?)